docs/guide/environments.md

```diff
@@ -5,7 +5,7 @@ You can configure several environments, such as production, development or stagi
 
 !!! tip
 
-    See also [the Symfony Recipe about "How to Master and Create new Environments"](http://symfony.com/doc/2.8/configuration/environments.html)
+    See also [the Symfony Recipe about "How to Master and Create new Environments"](http://symfony.com/doc/3.4/configuration/environments.html)
```

Comment:
```suggestion
    See also [the Symfony Recipe about "How to Master and Create new Environments"](http://symfony.com/doc/4.3/configuration/environments.html)
```

Reply:
Only 3.4 version of this section is marked as maintained.